reserved/persona.go

```diff
@@ -0,0 +1,51 @@
+package reserved
+
+import (
+	"log"
+	"math"
+
+	"github.com/wopta/goworkspace/models"
+)
+
+func personaReserved(policy *models.Policy) (bool, *models.ReservedInfo) {
+	log.Println("[personaReserved]")
+
+	reservedInfo := &models.ReservedInfo{
+		Reasons:       make([]string, 0),
+		RequiredExams: make([]string, 0),
+	}
+
+	isReserved := false
+	reason := "BMI index out of range"
+
+	if len(policy.Assets) == 0 || policy.Assets[0].Person == nil || policy.Assets[0].Person.Weight == 0 || policy.Assets[0].Person.Height == 0 {
+		return isReserved, reservedInfo
+	}
```

Comment:
we could enhance this control to make it more legible. One option is to make to variables that hold different kind of information as: policyHasAsset, and personIsInitialized (the naming is just an example, better ones would be an improvement)